Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -59,9 +60,15 @@
 #if HAVE(PHOTOS_UI_PRIVATE)
 SOFT_LINK_PRIVATE_FRAMEWORK(PhotosUIPrivate)
 SOFT_LINK_CLASS(PhotosUIPrivate, PUActivityProgressController)
+SOFT_LINK_CLASS(PhotosUIPrivate, PHPickerConfiguration)
+SOFT_LINK_CLASS(PhotosUIPrivate, PHPickerViewController)
+SOFT_LINK_CLASS(PhotosUIPrivate, PHPickerResult)
```

Comment:
None of these are in `PhotosUIPrivate`?

Reply:
This comment still needs to be addressed. It's strange that we are using `PhotosUIPrivate` here, when all of these classes are public API, in `PhotosUI`.